plugins/by-name/none-ls/prettier.nix

```diff
@@ -1,38 +1,41 @@
-{ lib, config, ... }:
+{
+  lib,
+  config,
+  options,
+  ...
+}:
 let
   cfg = config.plugins.none-ls.sources.formatting.prettier;
   ts-ls-cfg = config.plugins.lsp.servers.ts_ls;
+  opt = options.plugins.none-ls.sources.formatting.prettier;
+  defaultPrio = (lib.mkOptionDefault null).priority;
 in
 {
-  options.plugins.none-ls.sources.formatting.prettier = {
+  opt = {
```

Comment:
This is wrong. It gets a little confusing, but `opts` is a local variable we've created that gets its value from an attr in the `options` module argument.

Here we're _defining_ an `options` that is part of our module outputs.

This change makes it so that we actually define a literal `opts` attr instead of an `options` attr :sweat_smile:

Reply:
My bad!
_fixed_